Libplanet/Crypto/PublicKeyGetter.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+
+namespace Libplanet.Crypto
+{
+    /// <summary>
+    /// A static getter class for determine which public key it is by given <see cref="byte"/>
+    /// representation and returns a concrete class type in <see cref="IPublicKey"/>.
+    /// </summary>
+    public static class PublicKeyGetter
+    {
+        /// <summary>
+        /// <p>Gets a concrete class in <see cref="IPublicKey"/> by checking given <see cref="byte"/>
```

Comment:
```suggestion
        /// <p>Gets a concrete class in <see cref="IPublicKey"/> by checking given <see cref="byte"/>
```
Line length exceeds 100 characters 😢

Reply:
Oops, Amended again :P